app/upgrades/v16/upgrades.go

```diff
@@ -53,10 +53,9 @@ func CreateUpgradeHandler(
 			logger.Error("failed to migrate the fee collector", "error", err.Error())
 		}
 
-		// TODO: uncomment when ready
-		// if err := BurnUsageIncentivesPool(ctx, bankKeeper); err != nil {
-		//	logger.Error("failed to burn inflation pool", "error", err.Error())
-		// }
+		if err := BurnUsageIncentivesPool(ctx, bk); err != nil {
```

Comment:
Do we have test for this?

Reply:
yup: https://github.com/evmos/evmos/blob/428ad7440141963702cd7016bfa515360ecc3952/app/upgrades/v16/upgrades_test.go#L51-L69